src/patterns/dosdp-patterns/blood_vessel_pattern.yaml

```diff
@@ -0,0 +1,78 @@
+pattern_name: blood_vessel_pattern
+pattern_iri: http://purl.obolibrary.org/obo/uberon/patterns/blood_vessel_pattern.yaml
+description: "This is a pattern for a blood vessel."
```

Comment:
Its good to be a bid more verbose usually. But I am not insistant. What is the goal of this pattern, what kinds of blood vessels are covered and a hint at how.

Reply:
https://github.com/INCATools/dead_simple_owl_design_patterns/issues/67

See section "description should describe the pattern instances not the class instances" onwards, and successive comments

Reply:
Thank you, I will rewrite the description.